packages/stylelint-plugin/rules/index.js

```diff
@@ -0,0 +1 @@
+module.exports = [require('./content-no-strings')];
```

Comment:
This allows us to add multiple rules to this array and have them all be registered